src/core/execute.c

```diff
@@ -686,6 +686,28 @@ _printf_(2, 3) static int write_confirm_message(const char *vc, const char *form
         return 0;
 }
 
+static int write_confirm_error(int err, const char *vc) {
+        _cleanup_close_ int fd = -1;
+
+        assert(vc);
+        assert(err < 0);
+
+        fd = open_terminal(vc, O_WRONLY|O_NOCTTY|O_CLOEXEC);
+        if (fd < 0)
+                return fd;
+
+        if (err == -ETIMEDOUT)
+                dprintf(fd, "Confirmation question timed out, "
```

Comment:
I must admit that one thing with the current coding style I really disagree is those overly long lines (both for code and comments): IMHO they're definitely not more readable even if they would fit in the screen width...

I know that the hard limit of 80 chars is something obsolete and I'm not advocating for keeping it.

However  I usually have my screen splitted vertically with 2 buffers containing different source files which is my primarly usage of large screen and in that case comments those overly long lines doesn't fit anymore.

And even when the lines fit they're not easier to read for me. After all there're reasons why books use typography rules which have margins and never user such long lines.

My 2 cents.